toplevel/entity/entity.go

```diff
@@ -361,21 +364,36 @@ func createBaseExistingEntities(instanceAddr string) ([]entity, error) {
 				return nil, errors.New(fmt.Sprintf(
 					"Failed to convert element within `aliases` to map[string]interface{} for entity id: %s", id))
 			}
+
 			if _, exists := vals["id"]; !exists {
 				return nil, errors.New(fmt.Sprintf(
 					"Required `id` attribute not found on alias element for entity id: %s", id))
 			}
 			aliasId := vals["id"].(string)
+
 			if _, exists := vals["name"]; !exists {
 				return nil, errors.New(fmt.Sprintf(
-					"Required `name` attribute not found on alias element for entity-alias id: %s", id))
+					"Required `name` attribute not found on alias element for entity id: %s", id))
 			}
 			aliasName := vals["name"].(string)
+
+			mountType := ""
 			if _, exists := vals["mount_type"]; !exists {
-				return nil, errors.New(fmt.Sprintf(
-					"Required `mount_type` attribute not found on alias element for entity-alias id: %s", id))
+				accessor, exists := vals["mount_accessor"]
```

Comment:
the undocumented property which tells us if this is made by vault or made by us

Reply:
More or less. Not great documentation on it but from my testing, but `mount_type` does not always get set for Vault's underlying aliases, but `mount_accessor` is always set. I think hashicorp fixed this in a later version of vault and why we hadn't encountered it in our other environment.